### DISCUSSION TITLE
Best practice question for just using Kafka Connect functionality.

### DISCUSSION THREAD

**Discussion by jvieten** [strimzi/strimzi-kafka-operator#7955] [General]
Hello,
when only needing  to use Kafka-connect functionality , should I install all operators by for example applying 
[https://strimzi.io/install/latest?namespace=kafka](url) . Or is there is there a possibility  to only install a selected subset?

**Comment by scholzj** [comment]
Strimzi has 3 operators ->
* The Cluster operator for managing the Kafka server components. This is one operator which manages it all (ZooKeeper, brokers, Connect, Mirror Maker etc.). But you can use only some parts of it. E.g. only Connect and connect it to your own Kafka cluster not managed by Strimzi.
* The Topic and User operators which are managing Kafka users and topics. These are basically just part of the Strimzi Kafka cluster (i.e. brokers) deployment. You can install them separately if you want to use them with your own Kafka cluster. But typically, these are just part of the Strimzi Kafka cluster deployment.

`https://strimzi.io/install/latest?namespace=kafka` installs the Cluster Operator only. And yes, that is needed to use Strimzi's Kafka Connect features.
